src/app/admin-panel/_actions/create-equipment.ts

```diff
@@ -1,22 +1,25 @@
 "use server";
 
-import { getToken } from "@/auth/getToken";
+import { fetchPost } from "@/server-endpoints/fetchServer";
+import { createEquipmentSchema } from "@/schemas/equipmentSchemas";
 
 export async function createEquipment(formData: FormData) {
-  const token = await getToken();
-  if (!token) {
-    throw new Error("Not authenticated");
+  const name = formData.get("name") as string;
+  const id = formData.get("id") as string;
+
+  if (id) {
+    console.log("waiting for PUT /equipments/:id to be implemented");
+    // await fetch(`http://localhost:8080/equipments/${id}`, {
+    //   method: "PUT",
+    //   headers: {
+    //     "Content-Type": "application/json",
+    //   },
+    //   body: JSON.stringify({ name }),
+    // });
+    // return;
   }
 
-  await fetch("http://localhost:8080/equipments", {
-    method: "POST",
-    headers: {
-      "Content-Type": "application/json",
-      Authorization: `Bearer ${token}`,
-    },
-    body: JSON.stringify({
-      id: "hhbhb",
-      name: formData.get("name"),
-    }),
+  await fetchPost("http://localhost:8080/equipments", createEquipmentSchema, {
```

Comment:
Honestly I don't see a reason to really check the structure of what we get in response if we're not using it. Seems like a waste of resources here (If it were large scale app it would make a big difference)